pos_default_shipping_date/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1 @@
+* Christian Santamaría <christian.santamaria@solvos.es>
```

Comment:
Move this file and `DESCRIPTION.rst` to recommended `.md` extension, starting from v17. Once in `.md`, we need to escape these characters:
```suggestion
* Christian Santamaría \<christian.santamaria@solvos.es\>
```
Then, yo can re-create README files